server/models/User.ts

```diff
@@ -292,11 +290,12 @@ class User extends ParanoidModel {
   };
 
   updateSignedIn = (ip: string) => {
-    this.lastSignedInAt = new Date();
+    const now = new Date();
+    this.lastActiveAt = now;
```

Comment:
Ensure `lastActiveAt` is updated on signin, removes the need for additional updates elsewhere